### ISSUE TITLE
[BUG]  Spanish translation not working in some fields in "Glalie" template

### ISSUE THREAD

**Issue by NeilJared** [AmruthPillai/Reactive-Resume#1051]
I have Spanish translation working properly, but when I choose Glalie template, it does not translate the fields Phone Number and Email Address, which appear in English. All other templates get properly translated automatically.

BTW, will you add other templates? It will be really appreciated.

Kind regards,

fruizaviles

**Comment by mjesusoj** [comment]
Yes, I try to generate and download PDF with spanish version and on the page the field of date when it's (actually) but when I download PDF shows "Present" in English.

I think it's a bug and I don't know how to solve. Thank you for help!

**Comment by AmruthPillai** [comment]
This issue was automatically closed as it was created before 2024-01-01.
